web/src/stream_list.ts

```diff
@@ -839,6 +841,31 @@ export function set_event_handlers({
 }: {
     on_stream_click: (stream_id: number, trigger: string) => void;
 }): void {
+    $("#stream_filters").on(
+        "mouseenter",
+        "li .subscription_block .stream-name",
+        function (this: HTMLElement, e) {
+            const stream_id = stream_id_for_elt($(e.target).parents("li"));
+            const current_narrow_stream_id = narrow_state.stream_id();
+            const current_topic = narrow_state.topic();
+            if (!(current_narrow_stream_id === stream_id && current_topic)) {
+                return;
+            }
+            e.stopPropagation();
+            const $elem = $(this);
+            tippy.default(util.the($elem), {
+                showOnCreate: true,
+                onShow(instance) {
+                    instance.setContent($t({defaultMessage: "Go to channel feed"}));
+                },
+                onHidden(instance) {
+                    instance.destroy();
+                },
+                appendTo: () => document.body,
+            });
```

Comment:
Should this tooltip have delay? 

Also usually I'd expect us to write this using `tippy`'s own logic for deciding when to display it to do delays and the like; i.e. just declare a tooltip on channels in the left sidebar with a `delay` and an `onMount` function that does return false` if we don't want to show a tooltip.

Is there a reason you preferred this model?

Reply:
There isn't, I think I'm just not familiar enough with `tippy` yet. I can move to that.

Reply:
I'm not sure if this is possible actually, asked some questions [here on CZO](https://chat.zulip.org/#narrow/channel/6-frontend/topic/tippy.20prevent.20mount)